pointcloud/basic_octree_utils.go

```diff
@@ -183,3 +195,22 @@ func getCenterFromPcMetaData(meta MetaData) r3.Vector {
 func getMaxSideLengthFromPcMetaData(meta MetaData) float64 {
 	return math.Max((meta.MaxX - meta.MinX), math.Max((meta.MaxY-meta.MinY), (meta.MaxZ-meta.MinZ)))
 }
+
+func getMaxProb(octree *BasicOctree, p r3.Vector) (float64, error) {
```

Comment:
[major] since max Prob is just the data/100, isnt this jsut as simple as:
```
func getMaxProb(octree *BasicOctree, p r3.Vector) (float64, error) {
     d,err := octree.At(P.X, P.Y, P.Z)
    if err != nil {
         return 0, err
      }
     if !d.HasValue() {
         return 0, errors.New("no value in data")
    }
    return d.Value()/100., err
}
```

Reply:
Agree, but wouldn't it be `d.maxProb` instead?

Reply:
d is is a `Data` struct which does not store `maxProb`

Reply:
I'm confused then.  `d.Value()/100.` is only equal to `maxProb` when it's a leafNodeFilled, no?  Why is this correct?

Reply:
This would transform this function into a getter so its signature should be:

```
func (octree *BasicOctree) MaxProb() float64 {
        return octree.maxProb
}
```